spec/models/account_user_spec.rb

```diff
@@ -4,15 +4,15 @@
 
 RSpec.describe AccountUser, type: :model do
   context 'with existing account user' do
-    subject(:user) { FactoryBot.create(:account_user) }
+    subject(:user) { FactoryBot.create(:user) }
```

Comment:
subject should be of type `AccountUser` (since that is what we should be testing).